mage_ai/data_preparation/models/block/__init__.py

```diff
@@ -1190,16 +1192,21 @@ def get_outputs(
                 else:
                     row_count, column_count = data.shape
 
-                columns_to_display = data.columns.tolist()[:DATAFRAME_ANALYSIS_MAX_COLUMNS]
-                data = dict(
-                    sample_data=dict(
-                        columns=columns_to_display,
-                        rows=json.loads(data[columns_to_display].to_json(orient='split'))['data']
-                    ),
-                    shape=[row_count, column_count],
-                    type=DataType.TABLE,
-                    variable_uuid=v,
-                )
+                if csv_lines_only:
+                    data = data.to_csv(header=True, index=False).strip('\n')
```

Comment:
I think this changes the return value type of this the method. Could you either return the value in the format or update the method return value type?
It would be better to add method comments as well.

Reply:
Got it, somewhat related to the other comment, but didn't catch that it was a giant string. Updated the return value to be consistent with the return type. It returns a dictionary with a value that is a list of strings (pertaining to each row in the table).